mono/mini/method-to-ir.c

```diff
@@ -8519,7 +8519,11 @@ mono_method_to_ir (MonoCompile *cfg, MonoMethod *method, MonoBasicBlock *start_b
 						MONO_EMIT_NEW_BRANCH_BLOCK (cfg, OP_IBEQ, is_ref_bb);
 
 						/* Non-ref case */
-						nonbox_call = (MonoInst*)mini_emit_calli (cfg, fsig, sp, addr, NULL, NULL);
+						if (cfg->llvm_only)
```

Comment:
Shouldn't we just change `mini_emit_calli` to call `emit_llvmonly_calli` when `cfg->llvm_only` is true? There are a few other callsites in method-to-ir.c that don't use `emit_llvmonly_calli'.